01-lets_get_started/01.cpp

```diff
@@ -0,0 +1,35 @@
+//Code Compile and Run by codechef IDE
+
+#include <iostream>
+using namespace std;
+
+int main() 
+{
+  int size;
+  cout << "Enter Box Size\n";
+  cin >> size;
+
+  for(int col=0;col<size;++col)
```

Comment:
Consider putting spaces between operators, it may read better that way. For instance, the line:

`for(int col=0;col<size;++col)`

can be written as:

`for(int col = 0; col < size; ++col)`

Observe closely where there is space & where this no space (hint: there is no space near brackets or immediately before `;`)